DirectProgramming/DPC++FPGA/Tutorials/Features/ac_types/ac_fixed/README.md

```diff
@@ -0,0 +1,260 @@
+
+### Using the ac_fixed math functions
+
+To use this type in your code, you must include the following header:
+
+```cpp
+#include <sycl/ext/intel/ac_types/ac_fixed_math.hpp>
+```
+
+The flag `-qactypes` will ensure that the compiler includes the header and links against the necessary libraries for emulation of the math library functions.
+
+This tutorial design contains two kernels `CalculateWithFloat` and `CalculateWithACFixed`. Both calculate the simple expression:
+```cpp
+  square_root ( sine(x) * sine(x) + cosine(x) * cosine(x) )
+```
+for some input `x`.
+
+The kernel `CalculateWithFloat` uses floating point values and the standard math library while `CalculateWithACFixed` uses `ac_fixed` values and the `ac_fixed` math library. The `ac_fixed` inputs are instantiated with the following parameters:
+
+```cpp
+  W = 10, I = 3, S = true
+```
+
+Clearly, the `ac_fixed` numbers are smaller in size than floating point numbers. This results in reduction of the FPGA resources utilized by the functions at the expense of accuracy. To see the trade-offs between accuracy compare the numeric results of the operations. The area utilization differences will be discussed in the section on `Examining the reports`.
+
+When you use the `ac_fixed` library, keep the following points in mind:
+
+1. Input Bit Width and Input Value Range Limits
+
+    The fixed-point math functions have bit width and input value range requirements. All bit width and input value range requirements are documented at the top of the ac_fixed_math.hpp file. For example, the `sin_fixed` and `cos_fixed` functions require the integer part's bit width to be 3, and the input value range to be within [-pi, pi].
+
+2. Return Types
+
+    For fixed-point functions, each function has a default return type. Assigning the result to a non-default return type triggers a type conversion and can cause an increase in logic use or a loss of accuracy in your results.  All return types are documented at the top of the ac_fixed_math.hpp file. For example, for `sin_fixed` and `cos_fixed`, the input type is `ac_fixed<W, 3, true>`, and the output type is `ac_fixed<W-1, 2, true>`.
+
+3. Accuracy
+ - Floating point vs Fixed point
+
+    The host program (`main()` function) for this tutorial gives you an estimate of the difference between the correct result and the result provided by the math library functions. The floating point version (which has a greater bit width in this case) generates a more accurate result.
+ 
+ - Emulation vs Simulation for fixed point math operations
+
+    Due to the differences in the internal math implementations, the results from `ac_fixed` math functions in simulation and emulation might not always be bit-accurate. In this example you can observe the difference between emulation and simulation.
```

Comment:
When this tutorial releases, will simulation be supported? Is this limitation still present in oneAPI? Having a computational difference between emulation and hardware sounds like it will defeat the purpose of emulation :(